modules/javafx.controls/src/main/java/javafx/scene/control/ScrollPane.java

```diff
@@ -354,6 +354,8 @@ public final DoubleProperty vmaxProperty() {
      * If true and if the contained node is a Resizable, then the node will be
      * kept resized to match the width of the ScrollPane's viewport. If the
      * contained node is not a Resizable, this value is ignored.
+     *
+     * @defaultValue false
```

Comment:
This is unrelated to the fix, and you were otherwise not touching this file. Why change it?

Reply:
it is related to the fix, may be we differ in the _degree_ of relation.  I did not want to create a separate ticket for such a minor clarification...  but maybe I should, and fix all the missing `@defaultValue`s across the controls.

Reply:
Filed https://bugs.openjdk.org/browse/JDK-8341925